interfaces/Portalicious/src/app/pages/projects-overview/project-summary-card/project-summary-card.component.html

```diff
@@ -0,0 +1,110 @@
+<ng-template
+  #metric
+  let-value="value"
+  let-label="label"
+>
+  <dd class="px-2 py-1 font-bold">
+    @switch (metrics.status()) {
+      @case ('pending') {
+        <p-skeleton
+          class="my-1 cursor-progress text-lg"
+          height="1.125rem"
```

Comment:
> [!WARNING]
> See comment about `titlePortal`(Lines 34-48) first, before reading this.

These font-size/style classes are not exactly matching the finally loaded value/label size. So it would need some more manual tweaking.

But I'd like to have a more 'logical' way of styling the skeleton AND "the loaded value" all in 1 go...